### PR TITLE
docs(as): add missing attribute and computed behavior

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add missing attribute and notes.
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/resources/as_configuration.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ subcategory: "Auto Scaling"
 
 Manages an AS Configuration resource within HuaweiCloud.
 
+~> Make sure the configuration is not bound to a AS group when your resources are recreate by ForceNew behavior.
```
Comment (niuzhenguo) [inline docs/resources/as_configuration.md]: Nits: s/recreate/recreated

Comment (niuzhenguo) [reply docs/resources/as_configuration.md]: s/a AS group/an AS group

Comment (Lance52259) [reply docs/resources/as_configuration.md]: done.

**Pull request merged**